websites/A/atomicradio/presence.ts

```diff
@@ -0,0 +1,64 @@
+const presence = new Presence({
+		clientId: "904084297831571518",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/16lVdWy.png",
```

Comment:
Ditto (Logo's must be 512x512 as per [our guideliness](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).)